src/main/scala/chiseltest/legacy/backends/verilator/VerilatorBackend.scala

```diff
@@ -246,5 +251,15 @@ class VerilatorBackend[T <: Module](
 
       simApiInterface.finish() // Do this to close down the communication
     }
+    collectCoverageAnnotations()
```

Comment:
Nit: wondering if this should be `convertCoverageAnnotations` or similar to make clear it's distinction from other `collectCoverageAnnotations` methods

Reply:
OK. I will rename it.

Reply:
fixed